src/main/java/fi/metatavu/metamind/bot/MetamindBotConfiguration.java

```diff
@@ -1,77 +1,109 @@
 package fi.metatavu.metamind.bot;
-import java.util.ArrayList;
+
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
-import com.bladecoder.ink.runtime.Story;
-import com.rabidgremlin.mutters.bot.ink.CurrentResponse;
 import com.rabidgremlin.mutters.bot.ink.InkBotConfiguration;
 import com.rabidgremlin.mutters.bot.ink.InkBotFunction;
-import com.rabidgremlin.mutters.bot.ink.SessionUtils;
 import com.rabidgremlin.mutters.bot.ink.StoryUtils;
+import com.rabidgremlin.mutters.core.CompoundIntentMatcher;
 import com.rabidgremlin.mutters.core.Intent;
-import com.rabidgremlin.mutters.core.IntentMatch;
 import com.rabidgremlin.mutters.core.IntentMatcher;
-import com.rabidgremlin.mutters.core.session.Session;
 import com.rabidgremlin.mutters.opennlp.intent.OpenNLPIntentMatcher;
 import com.rabidgremlin.mutters.opennlp.intent.OpenNLPTokenizer;
 import com.rabidgremlin.mutters.opennlp.ner.OpenNLPSlotMatcher;
+import com.rabidgremlin.mutters.slots.LiteralSlot;
 import com.rabidgremlin.mutters.slots.NumberSlot;
+import com.rabidgremlin.mutters.templated.SimpleTokenizer;
+import com.rabidgremlin.mutters.templated.TemplatedIntent;
+import com.rabidgremlin.mutters.templated.TemplatedIntentMatcher;
 
 import opennlp.tools.tokenize.WhitespaceTokenizer;
 
 public class MetamindBotConfiguration implements InkBotConfiguration {
   
+  private List<InkBotFunction> functions;
+  
+  public MetamindBotConfiguration(List<InkBotFunction> functions) {
+    this.functions = functions;
+  }
+  
   @Override
   public IntentMatcher getIntentMatcher() {
-    // model was built with OpenNLP whitespace tokenizer
+
+    TemplatedIntentMatcher templatedIntentMatcher = new TemplatedIntentMatcher(new SimpleTokenizer());
+
+    TemplatedIntent templatedElectricyComsumptionIntent = templatedIntentMatcher.addIntent("GaveElectricityConsumption");
+    templatedElectricyComsumptionIntent.addUtterance("{ElectricityConsumption}");
+    templatedElectricyComsumptionIntent.addUtterance("{ElectricityConsumption} kwh");
+    templatedElectricyComsumptionIntent.addUtterance("{ElectricityConsumption}kwh");
+    templatedElectricyComsumptionIntent.addUtterance("{ElectricityConsumption} kilowattituntia");
+    templatedElectricyComsumptionIntent.addSlot(new NumberSlot("ElectricityConsumption"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ElectricityConsumption" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)